WordPress/gutenberg
# Reduce navigation sidebar width

Alternative to https://github.com/WordPress/gutenberg/pull/58442.

## What?
Reduce the width of the navigation sidebar in the site editor.

## Why?
On smaller screens it's almost comically large, and wastes a lot of space that is better used by the content frame.

<img width="1174" alt="Screenshot 2024-07-11 at 13 46 13" src="https://github.com/WordPress/gutenberg/assets/846565/865b2344-b9c1-4c62-b5f1-d8bb807d4f7d">


## How?
This PR reduces the width to 280px – the same width as the Inspector. There's obviously scope to tune this.

<img width="1288" alt="Screenshot 2024-07-11 at 13 56 49" src="https://github.com/WordPress/gutenberg/assets/846565/42dcfd4c-3b8e-4ccb-b559-3a3b85b39b2c">



## Testing Instructions
1. Open the Site Editor
2. Observe the narrower navigation sidebar


## Caveat
The one trade-off to this change is that the Styles panel feels more cramped. But given that's a smaller part of the experience with a somewhat unclear future this seems worthwhile to me.
